scripts/setupTypeScript.js

```diff
@@ -97,8 +97,13 @@ fs.writeFileSync(tsconfigPath, tsconfig)
 if (!argv[2]) {
   // Remove the script
   fs.unlinkSync(path.join(__filename))
-  // Remove the scripts folder
-  fs.rmdirSync(path.join(__dirname))
+  // Check if the scripts folder is empty
+  if (fs.readdirSync(path.join(__dirname)).length === 0) {
+    // Remove the scripts folder
+    fs.rmdirSync(path.join(__dirname))     // just in case, keep the folder, there might be others scripts present
+  } else {
+    console.log('"scripts" folder not deleted. Folder not empty')
+  }
```

Comment:
I think this check needs to have a bit more nuance, filter out files beginning with `.` and specifically this file  (right now it would never delete because there's at minimum 1 file in the folder)

and I think we can skip the comment, in a way deleting is the more surprising part vs not doing anything.

Reply:
Thanks a lot for your reply, I'm not at my computer right now, but I'll double check it, I tried it and it worked ok. On linux the readdirsync function didn't take into account the `.` and `..` pseudo files, if that's what you are referring to.
I didn't try with a hidden file though.

Reply:
Yeah, I was thinking about Mac users with `.DS_Store` files which end up everywhere 👍

Reply:
I don't have a mac to try, but I just check if, after deleting setupNode.js file, the only file left is DS_store, in which case I delete it. Otherwise, I just remove the setupNode.js script and leave everything else untouhched.
see: https://github.com/sveltejs/template/pull/147/commits/e50b4493cb591610da20275e10abc23fb2e67c40